cmd/keymasterd/main.go

```diff
@@ -147,6 +147,7 @@ type RuntimeState struct {
 const redirectPath = "/auth/oauth2/callback"
 const secsBetweenCleanup = 30
 const maxAgeU2FVerifySeconds = 30
+const secsBetweenDependencyChecks = 60
```

Comment:
Please make this configurable. Either a command-line flag or config file option.

Reply:
done